images/linux/scripts/tests/Tools.Tests.ps1

```diff
@@ -322,15 +316,25 @@ Describe "Containers" -Skip:(Test-IsUbuntu16) {
 }
 
 Describe "Node.js" {
-    $testCases = @("node", "grunt", "gulp", "webpack", "parcel", "yarn", "newman") | ForEach-Object { @{NodeCommand = $_} }
+    $testCases = @("node", "grunt", "gulp", "webpack", "parcel", "yarn", "newman", "netlify") | ForEach-Object { @{NodeCommand = $_} }
 
     It "<NodeCommand>" -TestCases $testCases {
         param (
             [string] $NodeCommand
         )
 
         "$NodeCommand --version" | Should -ReturnZeroExitCode
-    }   
+    }
+
+    Context "Vercel CLI" {
```

Comment:
we can move this test to line 319 with testcases

Reply:
Thank you, moved vercel test to testcases